app/screens/settings/notification_settings_auto_responder/notification_settings_auto_responder.js

```diff
@@ -65,6 +65,22 @@ export default class NotificationSettingsAutoResponder extends PureComponent {
         this.saveUserNotifyProps();
     }
 
+    componentDidMount() {
+        setTimeout(() => {
+            requestAnimationFrame(() => {
+                this.focus();
+            });
+        }, 500);
+    }
+
+    focus = () => {
+        this.autoresponderInput.focus();
+    };
+
+    autoresponderRef = (ref) => {
+        this.autoresponderInput = ref;
```

Comment:
can you change this to use [React.createRef](https://reactjs.org/docs/refs-and-the-dom.html)

Reply:
Done!